Assets/PlayerAssets/Scripts/Experience.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class Experience : MonoBehaviour {
+
+	public double exp;
+    public double toNextLevel;
+    public int lvl;
```

Comment:
try to use good descriptors, `lvl` is too generic of a variable, maybe do charLevel?